core-web/libs/edit-content/src/lib/utils/functions.util.ts

```diff
@@ -171,9 +171,12 @@ export const getFieldVariablesParsed = <T extends Record<string, string | boolea
  *       not valid JSON, an empty object will be returned.
  */
 export const stringToJson = (value: string) => {
-    if (value && isValidJson(value)) {
-        return JSON.parse(value);
+    if (!value) {
+        return {};
     }
 
-    return {};
+    // Add double quotes around keys
+    const json = value.replace(/(\w+):/g, '"$1":');
```

Comment:
There is this edge case that I'm not sure if we want to take care about

<img width="342" alt="Screenshot 2024-04-03 at 3 27 58 PM" src="https://github.com/dotCMS/core/assets/63567962/3fcdbff5-23da-4251-82c3-46128e6610e5">

Reply:
I'll remove that. Users should set a valid JSON in the field variable.